bsp/ls1bdev/.gitignore

```diff
@@ -0,0 +1 @@
+rtconfig.h
```

Comment:
Please keep .gitignore file.

Reply:
do you mean remove .gitignore in the patch?